image_to_gridmap_demo_custom_node:
  image_topic: "/image_publisher/image"
  resolution: 0.2
  map_frame_id: "map"
  min_height: -0.1042
  max_height: 4.1409

grid_map_visualization:
  grid_map_topic: /terrain_map
  grid_map_visualizations:
    - name: elevation_points
      type: point_cloud
      params:
        layer: elevation
    - name: elevation_grid
      type: occupancy_grid
      params:
        layer: elevation
        data_min: -0.1042
        data_max: 4.1409
    - name: flat_grid
      type: flat_point_cloud
      params:
        height: 0.0
